### DISCUSSION TITLE
GitHub Student Dev Pack Application Rejected for Invalid Reasons

### DISCUSSION THREAD

**Discussion by sr5434** [community/community#86500] [GitHub Education]
Below are the reasons why my application was rejected:

 - You are unlikely to be verified until you have completed your GitHub billing information with your full name exactly as it appears in your academic affiliation document. You do not have to add a payment method. You may need to log out and log back in to GitHub before re-applying.

 - The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.

 - The image you submitted does not appear to match the proof type you selected. If your original document is not in English, please take a photo of the original next to an English translation.
The last two reasons are invalid, my schools name was written on the ID and I said it was an ID. My schools name has a dash in the listing but not on the ID. How can I get accepted?

**Comment by AidenJamesYt** [comment]
Hey, I think i can help :)

First, iff you have a school email address, use it as your primary email on your GitHub account, as that may help with the application process :)

The first thing you mentioned, the billing information, is most likely one of the issues.  You need to complete your billing information. Heres how you can do it (can confirm these steps work):

Go to your GitHub settings and click on "Billing". Then enter your full name **exactly** as it appears on your academic affiliation document, and remember, you do **NOT** need to add a payment method! And for the cherry on top, log back into Github.

It also states that the image uploaded doesnt contain some of your schools info. Make sure that your schools full logo is shown, with the name, like [this logo](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS54jkxhnDGL2xxSLNsTOGeMDSzMwtwznXkp2qPfRCCSg&s)

Make sure that you selected the correct proof type (Sttudent ID) :)

**Reply by sr5434** [reply]
Thanks for helping out! It is indeed a school ID, but my school doesn’t include a logo on it. I updated my billing info and will retry. If it doesn’t work, is there a way for me to request manual review?

**Reply by dijanatatar** [reply]
I'm having the same issue. I keep getting rejected with the following reason:
> Please complete your GitHub billing information with your full name exactly as it appears in your academic affiliation document. You do not have to add a payment method. You may need to log out and log back in to GitHub before reapplying. If you have only a single legal name, enter it in both the first and last name fields.

And I double checked, my name matches the one in the certificate confirming I am employed as a teaching staff at the University. I already tried re-entering my billing information and logging out and back in again, but I keep getting rejected. I don't know what else to try?
